test/unit/test_interhost.py

```diff
@@ -84,7 +84,7 @@ def test_unequal_genomes_error(self):
         genomeA = makeTempFasta([('chr1', 'ATGCACGTACGTATGCAAATCGG'), ('chr2', 'AGTCGGTTTTCAG'),])
         genomeB = makeTempFasta([('first_chrom', 'GCACGTACGTATTTGCAAATC')])
         with self.assertRaises(Exception):
-            cm = interhost.CoordMapper(genomeA, genomeB)
+            cm = interhost.CoordMapper2Seqs(genomeA, genomeB)
```

Comment:
hm... maybe we should be raising a more specific exception than just Exception....